revision-1/lima/authors/12.conversation.md

```diff
@@ -0,0 +1,12 @@
+<|user|>
+Hvernig get ég vitað hvort einhver sé að ljúga að mér?
+<|system|>
+Til að sjá hvort einhver sé að segja satt eða að ljúga að þér eru nokkrir hlutir sem er hægt að hafa í huga:
+* Fylgstu með því hvort að það sem manneskjan segir sé samkvæmt sjálfu sér. Ef hún heldur einhverju fram núna sem passar ekki við það sem hún hafa áður haldið fram eða ef hún stangast á við sjálfa sig innan sömu samræðnanna gæti það bent til þess að hún sé að ljúga að þér.
+* Leitaðu eftir streituummerkjum. Lygarar eiga oft erfitt með að ljúga þegar þeir eru stressaðir. Fylgstu með því hvort hin manneskjan virðist stressuð. Er hún að fitla við eitthvað? Er hún að stama? Eða að forðast augnsamband? Þetta geta allt verið merki um að hún sé að ljúga að þér. Hafðu það samt í huga að sumir eru góðir lygarar og geta logið án sýnilegra svipbrigða eða hegðunarbreytinga. Þess vegna er mikilvægt að leita að mörgum ummerkjum um lygar.
+* Fylgstu með hvað manneskjan segir ekki. Ef hún er að ljúga að þér gæti hún sniðið framhjá hlutum eða sleppt því að leiðrétta þig ef þú heldur einhverju fram sem er ekki satt.
+* Sýndu eftirfylgni. Lygarar svara oft með stuttum svörum til að forðast að koma upp um sig. Fylgdu spurningunum þínum eftir með spurningum eins og “Af Hverju gerðirðu það?” eða “Hvernig gerðist það?” Og athugaðu hvort þeim fipast.
+* Spurðu spurninga sem þú veist svarið við. Þetta er frábær leið til að grípa lygara glóðvolgan. Ef þú veist svarið geturðu sagt “Ó svo það varst þú sem braust vasan ekki Kalli” og sjá hvort manneskjan leiðrétti þig.
+* Leitaðu að örsvipbrigðum. Örsvipbrigði er örstutt og ósjálfrátt svipbrigði sem afhjúpar raunverulegar tilfinningar manneskju. Fólk sem er að ljúga gæti sýnd örsvipbrigði sem bera merki kvíða eða samviskubits. Þú þarft að fylgjast mjög vandlega með því að þessi svipbrigði endast bara í örskotsstund.
```

Comment:
Örsvipbrigði eru ... ósjálfráð... afhjúpa ([svipbrigði er fleirtölu orð](https://bin.arnastofnun.is/leit/svipbrig%C3%B0i))

...gæti sýnt